.github/ISSUE_TEMPLATE/zui.md

```diff
@@ -20,14 +20,19 @@ You need to be logged into a Figma account to properly view the Figma content.
 
 ## Open questions
 
-## Possible implementations
+## Workflow
 
-Develop this using Storybook. We want all the design system components to be documented through their own Storybook stories.
-
-For reference, you can look at the already existing ZUI components created on the `undocumented/new-design-system` branch (not ZUI components that exist on `main`, as they are not part of the new design system.)
-
-## Git
+### Git
 
 The main git branch for the work on the new design system is `undocumented/new-design-system`. Unless otherwise instructed, do your work on a new branch branched off from this branch.
 
 Name your branch `issue-number/zui-name`, ex: `issue-928/zui-button` for a branch where work is done that is documented in the issue with number 928, where a button component is being made.
+
+### Storybook
+
+Use [Storybook](https://storybook.js.org/) to develop the new design system components. If you are not familiar with working with Storybook, please ask and Ziggi or someone else will be happy to introduce you!
+When you have checked out the branch `undocumented/new-design-system` (and, as always when checking out a branch just to be sure, run `yarn install`), run `yarn storybook` in the terminal. This starts Storybook locally, and should open your browser to `localhost:6006` where you see all the components. Note that you want to only look at the ones under the "New design system" headline.
+
+### Files
+
```

Comment:
Why a new folder (newDesignSystem)? I believe the other design system components are directly in sec/zui, are they not?

Reply:
as part of the "make things more accessible for volunteers" things i did yesterday, i put all the design system zui components in a folder with this name. It was confusing to instruct people to look at components for inspiration when there was no easy way to find which ones to be inspired by. 

I don't think this is something that should survive beyond the development phase, it's basically just to make instructions and navigations easier in this part of the work.

Reply:
I understand! I don't see this reflected in the code, but maybe I'm looking in the wrong place or maybe you haven't pushed it?

Is the purpose mainly to find the right source files, or to find the components in Storybook? I'm asking because I believe that categorizing them in storybook is possible without changing the directory structure.

If the purpose is to find the correct source files, maybe they could instead be linked from somewhere? I'm suggesting this because ideally, if we can avoid making changes in directory structure over time, I think that's generally preferrable. When we move files around, that makes it more difficult to do git forensics when fixing bugs etc.

This is not a huge deal. If you do think that the separate directly is the best way to go, I won't object. I just want to make sure we don't do it unnecessarily if there are other solutions that would solve the problems that you've identified.

Reply:
You can't see it reflected on this branch, because this is branched off from `main` bc i need it merged to main to be able to create issues with this template in github. you can see the changes to the file structure on `undocumented/new-design-system`

Yes, the storybook structure i have updated inside all storybook files, so on the `new-design-system` branch there are two categories in storybook, "new design system" and "old" (whatever we want the structur for storbyook to be needs to be thought through at some point). 

I am not sure i understand how to link all the files from one place, but if that's a preferred and easier way to collect them somewhere and point to them - let's do it :)

Reply:
I understand more clearly now, and after our discussion today I believe the conclusion was to just rename the folder to `components` and then we can keep that indefinitely. Will you make that change here before we merge?

Reply:
Done!